README.md

```diff
@@ -2,122 +2,179 @@ energy-blockchain
+docker build -t uploader:latest ami/ --no-cache
+```
+or just use
+```bash
+make
+```
+
+## API - Upload AMI Data
+### Data Types
+
++ Support Bems
+    + Carlab_BEMS
+    + SGESC_C_BEMS
+    + SGESC_D_BEMS
+    + ABRI_BEMS
+
++ Support Data Types
+    + bems_homepage_information(總用電)
+        |property|Type|Possible values|Description|
+        | ----- | ----- | ----- | ----- |
+        | id | string | `3d3d7ab1-3166-468f-8a70-d78cf844c1cc` | Identity of data |
+        | field | string | "NCKU" | 場域 |
+        | grid | float | 2.516 | 電網即時功率 |
+        | pv | float | 0.000 | 太陽能即時功率 |
+        | building | float | 2.466 | 大樓即時用電 |
+        | ess | float | -0.050 | 儲能即時功率 |
+        | ev | float | 0.000 | 充電樁即時功率 |
+        | updated_at | datetime | "2019-09-04T12:00:00" | %Y-%m-%dT%H:%M:%S |
+    + bems_ess_display(儲能系統)
+        |property|Type|Possible values|Description|
+        | ----- | ----- | ----- | ----- |
+        | id | string | `2a0d3a64-b813-4c39-8bbd-db4831e1d14b` | Identity of data |
+        | field | string | "NCKU" | 場域 |
+        | cluster | int | 1 | ESS編號 |
+        | power_display | float | -0.035 | 讀取實功輸出(kW)  |
+        | updated_at | datetime | "2019-09-04T12:00:00" | %Y-%m-%dT%H:%M:%S |
+    + bems_ev_display(充電樁)
+        |property|Type|Possible values|Description|
+        | ----- | ----- | ----- | ----- |
+        | id | string | `e827067a-3c41-4664-9950-c737138a32c5` | Identity of data |
+        | field | string | "NCKU" | 場域 |
+        | cluster | int | 1 | 充電柱編號 |
+        | power | float | 0.000 | 充電功率(kW) |
+        | updated_at | datetime | "2019-09-04T12:00:00" | %Y-%m-%dT%H:%M:%S |
+    + bems_pv_display(太陽光電)
+        |property|Type|Possible values|Description|
+        | ----- | ----- | ----- | ----- |
+        | id | string | `089c81a1-577b-4c09-90d8-21d3e69c1feb` | Identity of data |
+        | field | string | "NCKU" | 場域 |
+        | cluster | int | 1 | 太陽能編號 |
+        | PAC | float | 0.000 | 市電功率 |
+        | updated_at | datetime | "2019-09-04T12:00:00" | %Y-%m-%dT%H:%M:%S |
+    + bems_wt_display(中小型風力機)
+        |property|Type|Possible values|Description|
+        | ----- | ----- | ----- | ----- |
+        | id | string | `ce230f85-bddc-477d-8833-a5bf55a20254` | Identity of data |
+        | field | string | "NCKU" | 場域 |
+        | cluster | int | 1 | 風機編號 |
+        | WindGridPower | float | 0.000 | 風機功率 |
+        | updated_at | datetime | "2019-09-04T12:00:00" | %Y-%m-%dT%H:%M:%S |
 
 ### Usage
 
 curl
 
-`curl http://localhost:5000/{ems}/{type} -X POST --data @{json_file}`
-
-+ Parameters:
-
-    + ems : Type of ems. Should be `bems` or `hems`
+`curl http://localhost:4000/bems/upload -X POST --header 'Content-Type: application/json' --data @{json_file}`
 
-    + type : Type of upload data.
+Example for Sending AMI data:
 
-Example for Sending DR Event:
+`curl http://localhost:4000/bems/upload -X POST --header 'Content-Type: application/json' --data @upload.json`
```

Comment:
Discussion:
Put a sample JSON code block inside the readme will be much clearer for reader to understand (instead of splitting or opening new tab for comparison).
Your opinion?

Reply:
Fix in 26cea38.